### PR TITLE
Feature/emotionwheel

### PR DESCRIPTION
**Pull request by janaRicarda**
The emotionList is not yet responsive! it works only in mobile view and will be implemented in a further user story.
The onTouch event is for making it work on a real mobile phone.

### CODE REVIEW

File: SharedStyledComponents.js [hunk 1]
```diff
@@ -59,7 +59,8 @@ const StyledList = styled.ul`
 `;
 
 const StyledListItem = styled.li`
-  border: 1px solid var(--main-dark);
+  //border: 1px solid var(--main-dark);
```
Comment (F-Kirchhoff) [inline SharedStyledComponents.js:62]: please remove this line

File: components/Header.js [hunk 2]
```diff
@@ -52,11 +52,13 @@ const StyledHeader = styled.header`
...
 `;
 
-const StyledIconWrapper = styled(StyledWrapper)`
+const StyledIconWrapper = styled.article`
+  display: flex;
   justify-content: center;
   align-items: center;
   gap: 0.5rem;
   width: auto;
+  //border: 1px solid transparent;
```
Comment (F-Kirchhoff) [inline components/Header.js:61]: please remove this line and any other commented out code

File: components/Theme.js [hunk 3]
```diff
@@ -68,18 +98,37 @@ export const coldTheme = {
...
-  contempt: "#BAB5F7",
-  disgust: "#BAB5F7",
-  anger: "#BAB5F7",
+  // joy: "linear-gradient(to bottom left,#50cfe2, #9996fa)",
+  // surprise: "linear-gradient(to bottom left, #50cfe2, #9996fa)",
+  // fear: "linear-gradient(to bottom left, #50cfe2, #9996fa)",
+  // sadness: "linear-gradient(to bottom left, #50cfe2, #9996fa)",
+  // contempt: "linear-gradient(to bottom left, #50cfe2, #9996fa)",
+  // disgust: "linear-gradient(to bottom left, #50cfe2, #9996fa)",
+  // anger: "linear-gradient(to bottom left, #50cfe2, #9996fa)",
```
Comment (F-Kirchhoff) [inline components/Theme.js:111]: this as well

File: pages/_document.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ export default class MyDocument extends Document {
 
   render() {
     return (
-      <Html lang="en">
+      <Html style={{ overscrollBehavior: "none" }} lang="en">
```
Comment (F-Kirchhoff) [inline pages/_document.js:28]: please use the globalStyles for this you can target the html element:

```js
`
  html {
    overscroll-behavior: none
  }
`
```

**Review by F-Kirchhoff — APPROVED** [review]
Hey,
great PR! Please tidy up the code as described below. You can go directly to QA afterwards.

**Comment by janaRicarda** [comment]
Thank your for your review! I fixed everything what you have mentioned.

**Pull request merged**